app/src/main/java/com/example/cartoon/MainActivity.kt

```diff
@@ -265,6 +284,22 @@ class MainActivity : ComponentActivity() {
         return TensorImage.fromBitmap(bitmap)
     }
 
+    private fun hideSystemBars() {
+        window.decorView.systemUiVisibility = (
```

Comment:
hey @saurabhm13 . This change have to be here only when an app support android 4 and below. I believe whatever you are trying to do here can be done by styling the theme itself. [Reference](https://stackoverflow.com/a/25365193). 
Please try and let us know whether it is possible!